spec/loadbalancer/README.md

```diff
@@ -22,11 +20,65 @@ enterpriseconnect/loadbalancer:v1.1beta
 docker run -p 8080:80 --env-file $(pwd)/machine.env enterpriseconnect/loadbalancer:v1.1beta
 ```
 
+#### available tags
+- [```v1.1beta```](https://github.com/Enterprise-connect/oci/blob/v1.1beta/spec/loadbalancer/Dockerfile)
+
+#### tag usage
+- ```v1.1beta``` refers to the image to build agent ```#2730+```-relate releases.
```

Comment:
What does it to do with the agent and why? @ramaraosrikakulapu

Reply:
To run watcher, we have to run command ```./agent -cfg gw.yml -wtr``` which requires agent to download

Reply:
consider this- 

```v1.1beta``` is a beta release which is compatible with and only created for the EC agent in beta usage. It does not load balance traffic from any EC **production** releases. E.g. ```v1```, ```v1.1```, ```v1.2```, etc.

Reply:
The ```tag usage``` is still missing in the latest commit. @ramaraosrikakulapu